dashboard/app/views/plc/enrollment_unit_assignments/show.html.haml

```diff
@@ -18,7 +18,9 @@
       - module_assignment.plc_task_assignments.each do |task_assignment|
         - icon_class, icon_style = task_assignment.get_icon_and_style
         %i{class: 'fa ' + icon_class.to_s, style: icon_style}
-        = link_to task_assignment.plc_task.name, plc_enrollment_task_assignment_path(task_assignment)
+        = link_to task_assignment.plc_task.name,
+                  task_assignment.plc_task.try(:resource_url) ? "http://#{task_assignment.plc_task.resource_url}" : plc_enrollment_task_assignment_path(task_assignment),
+                  target: '_blank'
```

Comment:
It can - alternatively, can I just override the definition of plc_enrollment_task_assignment_path in one of my helpers, and put the logic there?